superset/views/core.py

```diff
@@ -242,16 +242,18 @@ def _deserialize_results_payload(
             return json.loads(payload)  # type: ignore
 
 
-class SliceFilter(SupersetFilter):
-    def apply(self, query, func):
+class SliceFilter(BaseFilter):
```

Comment:
Do you intend to inherit from `BaseFilter` here, or from `DatasourceFilter`, which is what used to be `SupersetFilter`?

Reply:
`BaseFilter` is still a better fit here, there is no much commonality between DatasourceFilter & SliceFilter